Automattic/jetpack
# Boost: update all cloud css on new post

Related: 76-gh-Automattic/boost-cloud

When updating a post or page, only Critical CSS for that post / page was being requested -- but our current Critical CSS State and Request objects make assumptions about always regenerating all of the Critical CSS for the site. Until we fix those issues, we can only realistically trigger a full re-gen for each change.

#### Changes proposed in this Pull Request:
* Regenerate all Critical CSS when a post/page is updated.

#### Jetpack product discussion
n/a

#### Does this pull request change what data or activity we track or use?
no

#### Testing instructions:
* Change a post / page
* Verify that cloud css updates everything automagically.
